nodes/arduino_handler.py

```diff
@@ -362,9 +371,12 @@ def process_message(line):
             actuator_state["heater_core_1_1"],
             actuator_state["chiller_compressor_1"]
         ).encode('utf-8')
-        serial_connection.write(message)
-        serial_connection.flush()
-        #rospy.logwarn("arduino_handler serial write {} bytes: >{}<".format(len(message),message.replace('\n','')))
+        try:
+            serial_connection.write(message)
+            serial_connection.flush()
+            trace('arduino_handler serial write %d bytes: >%s<', len(message), message.replace('\n',''))
+        except Exception as e:
```

Comment:
I think the exception you get here is `serial.serial_util.SerialException` we should be explicit as possible whenever possible.